### ISSUE TITLE
View F3 to calculate the "size" for DIR

### ISSUE THREAD

**Issue by yashpalgoyal1304** [doublecmd/doublecmd#793]
### Description

I only recently came to know about `<Space>` for calculating space (size) {nice pun hehe 😄} as well as for selection.

Due to this conflict, I do think that using `View F3` which doesn't do anything for folders would also be a good pick for this awesome functionality of calculating dir size.

References:
* mucommander
* [print directories] and `shift-alt-enter` for "Commands > Show Occupied Space" 

[print directories]: https://doublecmd.h1n.ru/viewtopic.php?f=5&t=2021&p=12246 "trail from searching with 'show dir size with view' in the linked mantis bug tracker for the doublecmd"

----------
P.S.:
The 'Alt F7' to open current folder in right panel (mentioned in answers in the forum post linked above) doesn't work in dc 1.0.10 (239) ??

**Comment by avmaksimov** [comment]
I agree. F3 is a good idea without selection. But ofter you need to sum some folder sizes so Space is more useful.

But also I suggest Esc as "Unmark all" for "Files panel".

Of course we can do it by set some options but as default it would better for new users.

**Comment by PetbkA** [comment]
Isn't `Ctrl+L` hotkey about it? You can select any files you want → Press the hotkey → Summary size of these files and dirs will be shown

**Comment by alexx2000** [comment]
Yes, `Ctrl+L` is a standard hotkey. `F3` already binded as `open` when called on a folder but broken in current version. `View` in case of a folder - show folder content.
